tests/validation_tests/round_four_specific_validations/test_tf_r4_specific_validations.py

```diff
@@ -752,3 +754,38 @@ def test_validate_funding_spent_FHSF(mocker, allocated_funding):
             row_indexes=[43, 71, 99],
         )
     ]
+
+
+def test_validate_funding_profiles_funding_secured_not_null():
+    funding_df = pd.DataFrame(
```

Comment:
Can this test also simulate two melted rows so it covers the dropping of duplicate indexes